rollkit/cmd/init.go

```diff
@@ -0,0 +1,123 @@
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/cometbft/cometbft/crypto"
+	cmtjson "github.com/cometbft/cometbft/libs/json"
+	cmprivval "github.com/cometbft/cometbft/privval"
+	cmttypes "github.com/cometbft/cometbft/types"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	configchain "github.com/ignite/cli/v28/ignite/config/chain"
+	"github.com/ignite/cli/v28/ignite/pkg/cliui"
+	"github.com/ignite/cli/v28/ignite/pkg/cliui/colors"
+	"github.com/ignite/cli/v28/ignite/services/chain"
+)
+
+func NewRollkitInit() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "init",
+		Short: "Init rollkit support",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			session := cliui.New()
+			defer session.End()
+
+			appPath, err := cmd.Flags().GetString(flagPath)
+			if err != nil {
+				return err
+			}
+			absPath, err := filepath.Abs(appPath)
+			if err != nil {
+				return err
+			}
+
+			rc, err := chain.New(absPath, chain.CollectEvents(session.EventBus()))
+			if err != nil {
+				return err
+			}
+
+			if localDa, err := cmd.Flags().GetBool(flagLocalDa); err == nil && localDa {
+				igniteConfig, err := rc.Config()
+				if err != nil {
+					return err
+				}
+
+				igniteConfig.Validators[0].Bonded = "1000000000stake"
```

Comment:
May we use a flag for the bonded and balance and use these values as minimum

Reply:
This value is expected by local-da, changing it will break it.